### DISCUSSION TITLE
Modbus Simulator enviromnent for Fuzzing algorithm

### DISCUSSION THREAD

**Discussion by AhmedAldysty** [pymodbus-dev/pymodbus#1955] [Q&A] [answered]
Hello 
Please, I need to develop a Modbus protocol as a test target for my Fuzzing method. Can someone guide me on how I can do this?
Regards

**Comment by janiversen** [comment]
Well you do not need to develop the modbus protocol, just use pymodbus, and add your fuzzy on top.

**Reply by AhmedAldysty** [reply]
Could you please provide more explanation?

**Reply by janiversen** [reply, marked as answer]
Please read the documentation and look at the examples.

As I really do not know what you want to do, I could only tell you that pymodbus implement the modbus protocol so no need for you to do that.
